library/core/src/result.rs

```diff
@@ -606,7 +608,12 @@ impl<T, E> Result<T, E> {
     #[must_use]
     #[inline]
     #[stable(feature = "is_some_and", since = "1.70.0")]
-    pub fn is_ok_and(self, f: impl FnOnce(T) -> bool) -> bool {
+    #[rustc_const_unstable(feature = "const_result_callback_methods", issue = "67792")]
```

Comment:
This should get a new tracking issue rather than reusing the huge one

Reply:
Not sure what would be the process for that. Can/should I do that or someone from the team is expected to?

Reply:
You can open one :) there's a template, just copy the changed function definitions there.